### PR TITLE
Lost Connection check 

### LINKED ISSUE #229: proxy command foreign key check
exception with foreign key checks

```
Exception 'yii\db\Exception' with message 'Error while sending QUERY packet. PID=352
The SQL being executed was: SET FOREIGN_KEY_CHECKS = 1;'

in /app/vendor/yiisoft/yii2/db/Schema.php:664

Stack trace:
#0 /app/vendor/yiisoft/yii2/db/Command.php(1263): yii\db\Schema->convertException(Object(yii\base\ErrorException), 'SET FOREIGN_KEY...')
#1 /app/vendor/yiisoft/yii2/db/Command.php(1075): yii\db\Command->internalExecute('SET FOREIGN_KEY...')
#2 /app/vendor/luyadev/luya-module-admin/src/proxy/ClientTable.php(181): yii\db\Command->execute()
#3 /app/vendor/luyadev/luya-module-admin/src/proxy/ClientTable.php(138): luya\admin\proxy\ClientTable->cleanup('ONLY_FULL_GROUP...')
#4 /app/vendor/luyadev/luya-module-admin/src/proxy/ClientTransfer.php(41): luya\admin\proxy\ClientTable->syncData()
#5 /app/vendor/luyadev/luya-module-admin/src/commands/ProxyController.php(205): luya\admin\proxy\ClientTransfer->start()
#6 [internal function]: luya\admin\commands\ProxyController->actionSync()
#7 /app/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#8 /app/vendor/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams(Array)
#9 /app/vendor/yiisoft/yii2/console/Controller.php(148): yii\base\Controller->runAction('', Array)
#10 /app/vendor/yiisoft/yii2/base/Module.php(528): yii\console\Controller->runAction('', Array)
#11 /app/vendor/yiisoft/yii2/console/Application.php(180): yii\base\Module->runAction('admin/proxy', Array)
#12 /app/vendor/yiisoft/yii2/console/Application.php(147): yii\console\Application->runAction('admin/proxy', Array)
#13 /app/vendor/yiisoft/yii2/base/Application.php(386): yii\console\Application->handleRequest(Object(yii\console\Request))
#14 /app/vendor/luyadev/luya-core/base/Boot.php(205): yii\base\Application->run()
#15 /app/vendor/luyadev/luya-core/bin/luya(22): luya\base\Boot->applicationConsole()
```

### PR DESCRIPTION
**Pull request by boehsermoe**
Check connection before cleanup. closes #229

### CODE REVIEW

File: src/proxy/ClientTable.php [hunk 1]
```diff
@@ -177,6 +178,12 @@ protected function prepare()
      */
     protected function cleanup($sqlMode)
     {
+        try {
+            Yii::$app->db->createCommand('SELECT CONNECTION_ID()')->execute();
```
Comment (nadar) [inline src/proxy/ClientTable.php]: @boehsermoe is this command safe for other dbms systems like:
- MySQL
- MariaDB
- SQLite
- PostgreSQL
- CUBRID
- Oracle
- MSSQL

Comment (nadar) [reply src/proxy/ClientTable.php]: If not, maybe we should make another DAO valid task to check whether connection works?

Comment (nadar) [reply src/proxy/ClientTable.php]: @boehsermoe maybe we could also add this command into the schema object check?

```
if ($this->db->schema instanceof \yii\db\mysql\Schema) {
```

Comment (boehsermoe) [reply src/proxy/ClientTable.php]: CONNECTION_ID only available in MySQL and MariaDB. It looks difficulty to handle all other systems.

**Pull request merged**